src/fragments/lib-v1/graphqlapi/existing-resources.mdx

```diff
@@ -18,9 +18,18 @@ Existing AWS AppSync resources can be used with the Amplify Libraries by referen
 }
 ```
 
+<InlineFilter filters={['swift','android',]}>
```

Comment:
I am not seeing a difference between flutter content and swift/android. Can this be reverted?